src/app/modules/shared/components/details-fields/details-fields.component.spec.ts

```diff
@@ -443,6 +443,32 @@ describe('DetailsFieldsComponent', () => {
     expect(component.projectSelected.emit).toHaveBeenCalledWith(data);
   });
 
+  it('on Selected Start Date should change End Date', () => {
+    const expectedStartDate = '2020-02-05';
+
+    component.onStartDateChange(expectedStartDate);
+    fixture.detectChanges();
+    const endDateInput: HTMLInputElement = fixture.debugElement.nativeElement.querySelector('#end_date');
+
+    expect(endDateInput.value).toEqual(expectedStartDate);
+  });
+
+  it('on Selected End Date should not change Start Date', () => {
+    const expectedStartDate = '2020-02-05';
+    const expectedEndDate = '2020-02-06';
+
+    component.ngOnInit();
+    fixture.detectChanges();
+    const startDateInput: HTMLInputElement = fixture.debugElement.nativeElement.querySelector('#start_date');
+    const endDateInput: HTMLInputElement = fixture.debugElement.nativeElement.querySelector('#end_date');
+    startDateInput.value = expectedStartDate;
+    endDateInput.value = expectedEndDate;
+    endDateInput.dispatchEvent(new Event('#end_date'));
+
+    expect(endDateInput.value).not.toEqual(startDateInput.value);
+  });
```

Comment:
Maybe you could add another assert where the startDateInput is equal to the `expectedStartDate `, so you be sure that the start date value didn't change.